File: app/src/main/java/com/song/deviceinfo/utils/TimeUtils.java
package com.song.deviceinfo.utils;

import android.annotation.SuppressLint;

import java.text.SimpleDateFormat;
import java.util.Calendar;
import java.util.Date;

/**
 * Created by chensongsong on 2020/9/22.
 */
public class TimeUtils {

    @SuppressLint("SimpleDateFormat")
    public static String formatDate(long time) {
        return new SimpleDateFormat("yyyy-MM-dd HH:mm:ss").format(new Date(time));
    }

    private void test(){
        Calendar calendar = Calendar.getInstance();
    }

}

File: app/src/main/java/com/song/deviceinfo/utils/TimeUtils.kt
package com.song.deviceinfo.utils

import android.annotation.SuppressLint
import java.text.SimpleDateFormat
import java.util.Calendar
import java.util.Date

/**
 * Created by chensongsong on 2020/9/22.
 */
class TimeUtils {
    private fun test() {
        val calendar = Calendar.getInstance()
    }
    
    companion object {
        @JvmStatic
        @SuppressLint("SimpleDateFormat")
        fun formatDate(time: Long): String {
            return SimpleDateFormat("yyyy-MM-dd HH:mm:ss").format(Date(time))
        }
    }
}
